app.js

```diff
@@ -10,10 +10,20 @@ function Projects (projs) {
   this.completedOn = projs.completedOn;
```

Comment:
**TODO FOR LOST POINTS**

Just putting a top level comment here that you should have some placeholder images somewhere so you can get an idea of what how images will look on your portfolio. Here's a tool I like to use for https://placehold.it/placeholders.